test/integration/test_server.cpp

```diff
@@ -1796,7 +1853,7 @@ TEST_CASE("WriteBatchHistory upsert and delete", "[integration][write-batch]") {
 
     request.add_earliest_start_files(TEST_RESOURCES_DIR +
                                      "books_history_earliest.csv");
-    request.add_update_files(TEST_RESOURCES_DIR + "books_history_upsert.csv");
```

Comment:
Will update files now be untested in history mode? Should there be another test section / test?